services/espnet-tts-fr/src/app.py

```diff
@@ -82,7 +107,23 @@ def segment_tts():
     try:
         totalWav = None
         durations = []
-        wavs = [tts(segment) for segment in data["segments"]]
+        wavs = []
+        for segment in data["segments"]:
+            # detect numerical in segments
+            segment_new = []
+            logger.debug(segment.split())
+            for s in segment.split():
+                try:
+                    logger.debug(f'Performing on: "{s}"')
```

Comment:
This is going to get _very_ chatty very quickly...

Reply:
Not merging right now, but do you anticipate this being useful going forward? I'm a little worried about the slim possibility of this leaking some personal info.

Reply:
I find this helpful to debug what component the TTS couldn't convert to numbers sometimes, but it can definitely be case specific (log in the `except` statement) rather than logging out everything. 

You might be right... In that case, I would have to change the logs of `multilang-support` as well because it has been logging the whole segment out everytime, both in English and French 🫤

Reply:
Ah, yeah please do that too then!

Reply:
Done that!